workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/containers/SideMenu/index.jsx

```diff
@@ -41,11 +41,12 @@ const SideMenu = () => {
 		() =>
 			subscriptionGroups?.filter(
 				(subscriptionGroup) => {
+					
 					if (subscriptionGroup.name === MENU_TYPES.liferayPaaS) {
```

Comment:
Does LiferaySaas need something like this too? Do you know why Paas needs this?

Reply:
I was looking at this [commit](https://github.com/liferay-customer-solutions/liferay-portal/commit/825ec3216e9510e0f3091e2bed9a5f9dbe1e1819) and because of Liferay PaaS and SaaS are not into the Product Activation tab anymore we don't need to check if PaaS and SaaS subscription group has activation. 
After this commit, the validation **hasLiferayPaasActivation** were added, but it just checks if Liferay PaaS is into the subscriptiongroups array and not checking for activation.

So after this commit, we check that twice for Liferay PaaS that we can see on line 148:

`hasLiferayPaasActivation && hasProductSubscription(PRODUCT_TYPES.dxpCloud)`

Both **hasLiferayPaasActivation** and **hasProductSubscription(PRODUCT_TYPES.dxpCloud)** look for paas in the subscriptionGroups array.

So IMO we don't need something like that for Liferay SaaS and we can consider removing hasLiferayPaasActivation

Reply:
and we already use hasProductSubscription to check for Liferay SaaS

Reply:
> we can consider removing hasLiferayPaasActivation

if we don't need it, can we remove it then. lets try to make the SaaS and PaaS logic as consistent as possible since they should work basically the same, albeit with different products.